### PR TITLE
MC-11016: Azure public ip addresses doc

### PR DESCRIPTION
**Pull request by franzpgarcia**
Docs for azure public ip addresses

### CODE REVIEW

File: source/includes/azure/_public_ip_addresses.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/publicipaddresses/:id</code>
+
+Retrieve a public IP address in a given [environment](#administration-environments)
+
+Attributes | &nbsp;
+------- | -----------
+`id` <br/>*string* | The id of the public IP address. This is a canonized id from azure which is the form of `/subscriptions/:subscriptionid/resourceGroups/:resourcegroup/providers/`Microsoft.Network/publicIPAddresses/:publicIpName
+`name` <br/>*string* | The name of the public IP address
+`state` <br/>*string* | The state of the public IP address. Possible state: `ATTACHED`, `DETACHED`
+`ipAddress` <br/>*string* | The IP address of the public IP. If the public IP has never been associated before, then you won't have an ip address yet.
```
Comment (lamminade) [inline source/includes/azure/_public_ip_addresses.md:94]: only one of these ends in a period

File: source/includes/azure/_public_ip_addresses.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/publicipaddresses</code>
+
+Retrieve a list of all public IP addresses in a given [environment](#administration-environments)
+
+Attributes | &nbsp;
+------- | -----------
+`id` <br/>*string* | The id of the public IP address. This is a canonized id from azure which is the form of `/subscriptions/:subscriptionid/resourceGroups/:resourcegroup/providers/`Microsoft.Network/publicIPAddresses/:publicIpName
+`name` <br/>*string* | The name of the public IP address
+`state` <br/>*string* | The state of the public IP address. Possible state: `ATTACHED`, `DETACHED`
+`ipAddress` <br/>*string* | The IP address of the public IP. If the public IP has never been associated before, then you won't have an ip address yet.
```
Comment (lamminade) [inline source/includes/azure/_public_ip_addresses.md:48]: also period inconsistency here

**Review by lamminade — APPROVED** [review]

File: source/includes/azure/_public_ip_addresses.md (hunk 2 above)
Comment (franzpgarcia) [reply source/includes/azure/_public_ip_addresses.md:48]: All fixed  :)

File: source/includes/azure/_public_ip_addresses.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+Retrieve a list of all public IP addresses in a given [environment](#administration-environments)
+
+Attributes | &nbsp;
+------- | -----------
+`id` <br/>*string* | The id of the public IP address. This is a canonized id from azure which is the form of `/subscriptions/:subscriptionid/resourceGroups/:resourcegroup/providers/`Microsoft.Network/publicIPAddresses/:publicIpName.
+`name` <br/>*string* | The name of the public IP address.
+`state` <br/>*string* | The state of the public IP address. Possible state: `ATTACHED`, `DETACHED`.
+`ipAddress` <br/>*string* | The IP address of the public IP. If the public IP has never been associated before, then you won't have an ip address yet.
+`region` <br/>*string* | The region where the public IP address is located.
+`sku`  <br /> *string* | The sku of the public IP.
```
Comment (fyrnaga) [inline source/includes/azure/_public_ip_addresses.md]: It would be nice to be the possible values, Standard/Basic

Comment (franzpgarcia) [reply source/includes/azure/_public_ip_addresses.md]: done

**Review by fyrnaga — APPROVED** [review]
LGTM

**Pull request merged**